File: problemas/matematica/1323/1323.cs
using System;

class URI {
    static int[] F = new int[101];
    
    static int Feynman(int n){
        if(F[n] == 0){
            F[n] = Feynman(n - 1) + n * n;
        }
        return F[n];
    }
    
    static void Main(string[] args) {
        string entrada;
        
        F[1] = 1;
        
        while((entrada = Console.ReadLine()) != null){
            int N = int.Parse(entrada);
            
            if(N == 0){
                break;
            }
            
            Console.WriteLine(Feynman(N));
        }
    }
}

File: problemas/matematica/1323/1323.jl
using Printf

F = [-1 for i in 1:101]
F[1] = 1

function feynman(n)
  if(F[n] == -1)
    global F[n] = feynman(n - 1) + (n * n);
  end
  return F[n]
end

while true
  n = parse(Int32, readline())

  if n == 0
    break
  end

  @printf "%d\n" feynman(n)
end